python/dgl/sparse/sparse_matrix.py

```diff
@@ -479,6 +479,128 @@ def is_diag(self):
         """Returns whether the sparse matrix is a diagonal matrix."""
         return self.c_sparse_matrix.is_diag()
 
+    def rowwise_select(self, ids: torch.Tensor):
+        """Returns a row-wise sparse matrix selected by ids.
+
+        Parameters
+        ----------
+        ids : tensor.Tensor
+            The indices indicate the rows selected from matrix.
+
+        The function has not support autograd.
+
+        Returns
+        -------
+        SparseMatrix
+            The row-wise sparse matrix only contains the rows selected.
+
+        Examples
+        --------
+        >>> indices = torch.tensor([0, 1, 1, 2, 3, 4], [0, 2, 4, 3, 5, 0]])
+        >>> val = torch.tensor([0, 1, 2, 3, 4, 5])
+        >>> A = dglsp.spmatrix(indices, val)
+        >>> row_ids = torch.tensor([0, 1, 4])
+        >>> B = A.rowwise_select(row_ids)
+        >>> B
+        SparseMatrix(indices=tensor([[0, 1, 1, 2],
+                                     [0, 2, 4, 0]]),
+                     values=tensor([0, 1, 2, 5]),
+                     shape=(3, 6), nnz=4)
+        """
+        return SparseMatrix(self.c_sparse_matrix.rowwise_select(ids))
```

Comment:
Does this exist?
If not, just raise an not implemented error.

Reply:
Now all APIs have implemented, please check.